rbd/mirror_desc_status_test.go

```diff
@@ -0,0 +1,164 @@
+//go:build !nautilus && ceph_preview
+// +build !nautilus,ceph_preview
+
+// IMPORTANT - when removing ceph_preview from this file also delete
+// rbd/mirror_desc_status_test.go as it will no longer serve a purpose.
```

Comment:
I hope you meant _rbd/mirror_stub_test.go_ here.

Reply:
Fixed.